docs/getting-started/throwing-errors.md

```diff
@@ -212,7 +212,7 @@ Your error codes should uniquely identify a _type_ of error your app can throw.
 
 ### Adding more data
 
-It's also sometimes useful to add extra details to an error so that further debugging of them can be done once we've determined what _type_ of error they are using `code`. Useful information might be the actual value of any input which caused an error, or further details about application state (e.g. the ID of the user this error occurred for).
+It's also sometimes useful to add extra details to an error so that further debugging of them can be done once we've determined what _type_ of error they are using via the `code` value. Useful information might be the actual value of any input which caused an error, or further details about application state (e.g. the ID of the user this error occurred for).
```

Comment:
__question:__ did you mean to delete the "using" here to read "once we've determined what type of error they are via the code value"?

Reply:
I don't think I have removed "using"?

The edited sentence reads:
> It's also sometimes useful to add extra details to an error so that further debugging of them can be done once we've determined what _type_ of error they are using via the `code` value.